src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -687,24 +686,25 @@ bool Windward(Real64 const CosTilt,      // Cosine of the surface tilt angle
     return AgainstWind;
 }
 
-bool SetAdaptiveConvectionAlgoCoefficient(EnergyPlusData &state,
-                                          const std::unordered_map<std::string, int> &HcInt_ConvectionTypesMap,
-                                          int *const InsideFaceAdaptiveConvectionAlgoParam,
-                                          const std::string equationName,
-                                          const std::string curveName,
-                                          const std::string sourceFieldName,
-                                          const std::string curveFieldName,
-                                          const std::string RoutineName,
-                                          const std::string CurrentModuleObject)
+int SetInsideAdaptiveConvectionAlgo(EnergyPlusData &state,
+                                    const std::unordered_map<std::string, int> &HcInt_ConvectionTypesMap,
+                                    bool &ErrorsFound,
+                                    const std::string &equationName,
+                                    const std::string &curveName,
+                                    const std::string &sourceFieldName,
+                                    const std::string &curveFieldName,
+                                    const std::string &RoutineName,
+                                    const std::string &CurrentModuleObject)
```

Comment:
This was being used to get input for the adaptive convention models but was broken for the external convection algorithms. See issue #8861. This updates it to have different functions for the inside and outside algorithms, and updates the signature to work a little better.